Lib/Common/Blob/BlobWriteStreamBase.cs

```diff
@@ -45,7 +45,7 @@ internal abstract class BlobWriteStreamBase :
         protected AccessCondition accessCondition;
         protected BlobRequestOptions options;
         protected OperationContext operationContext;
-        protected CounterEvent noPendingWritesEvent;
+        protected AsyncCounterEvent noPendingWritesEvent;
```

Comment:
I don't think this will build for all the platforms we support - I think we'll need to either change references to this in Lib/ClassLibraryCommon as well as in Lib/WindowsRuntime, or move the events into the respective derived classes (can't wait until we can merge the two codebases together)